EmbeddedSocialClient/sdk/src/main/java/com/microsoft/embeddedsocial/account/AccountDataStorage.java

```diff
@@ -50,6 +52,31 @@ static void store(Context context, AccountData accountData) {
 			.apply();
 	}
 
+	/**
+	 * Saves the current user's hashed email to the disk.
+	 * @param context runtime context
+	 * @param email hash of email
+	 */
+	static void storeEmail(Context context, String email) {
+		SharedPreferences prefs = getPrefs(context);
+		prefs.edit()
+				.putString(EMAIL, email)
```

Comment:
consider renaming this to HASHED_EMAIL so that it is clear that we are not storing emails in plaintext.

Reply:
:+1: